node/src/database/db_migrations.rs

```diff
@@ -2031,4 +2210,159 @@ mod tests {
+         values (\"0xD7d1b2cF58f6500c7CB22fCA42B8512d06813a03\",56784545484899,11111,null)");
+        insert_value(
+            &*pre_db_conn,
+            "insert into receivable (wallet_address, balance, last_received_timestamp) \
+             values (\"0xD2d1b2eF58f6500c7ae22fCA42B8512d06813a03\",-56784,22222)",
+        );
+        insert_value(&*pre_db_conn,"insert into pending_payable (transaction_hash, amount, payable_timestamp,attempt, process_error) \
+         values (\"0xb5c8bd9430b6cc87a0e2fe110ece6bf527fa4f222a4bc8cd032f768fc5219838\",9123,33333,1,null)");
+        let mut persistent_config = PersistentConfigurationReal::from(pre_db_conn);
+        let old_rate_pack_in_gwei = "44|50|20|32".to_string();
+        persistent_config
+            .set_rate_pack(old_rate_pack_in_gwei.clone())
+            .unwrap();
+
+        let conn = subject
+            .initialize_to_version(
+                &dir_path,
+                7,
+                false,
+                DbInitializationConfig::create_or_migrate(make_external_migration_parameters()),
+            )
+            .unwrap();
+
+        assert_table_created_as_strict(&*conn, "payable");
+        let expected_key_words: &[&[&str]] = &[
+            &["wallet_address", "text", "primary", "key"],
+            &["balance_high_b", "integer", "not", "null"],
+            &["balance_low_b", "integer", "not", "null"],
+            &["last_paid_timestamp", "integer", "not", "null"],
+            &["pending_payable_rowid", "integer", "null"],
+        ];
+        assert_create_table_stm_contains_all_parts(&*conn, "payable", expected_key_words);
+        //we don't need the strict constrain necessarily here
+        let expected_key_words: &[&[&str]] = &[
+            &["transaction_hash", "text", "not", "null"],
+            &["amount_high_b", "integer", "not", "null"],
+            &["amount_low_b", "integer", "not", "null"],
+            &["payable_timestamp", "integer", "not", "null"],
+            &["attempt", "integer", "not", "null"],
+            &["process_error", "text", "null"],
+        ];
+        assert_create_table_stm_contains_all_parts(&*conn, "pending_payable", expected_key_words);
+        assert_table_created_as_strict(&*conn, "receivable");
+        let expected_key_words: &[&[&str]] = &[
+            &["wallet_address", "text", "primary", "key"],
+            &["balance_high_b", "integer", "not", "null"],
+            &["balance_low_b", "integer", "not", "null"],
+            &["last_received_timestamp", "integer", "not", "null"],
+        ];
+        assert_create_table_stm_contains_all_parts(conn.as_ref(), "receivable", expected_key_words);
+        ["payable", "receivable", "pending_payable"]
+            .iter()
+            .for_each(|table_name| {
+                assert_table_does_not_exist(&*conn, &format!("_{}_old", table_name))
+            });
+        assert_values_still_there_plus_pair_of_integer_columns(&*conn, "payable", |row| {
```

Comment:
It seems to me that if you have these assertions--which are good!--you don't need the assertions inside `assert_create_table_stm_contains_all_parts()`.